staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -138,25 +136,40 @@ type ValidatingAdmissionPolicySpec struct {
 	// +optional
 	AuditAnnotations []AuditAnnotation `json:"auditAnnotations,omitempty" protobuf:"bytes,5,rep,name=auditAnnotations"`
 
-	// TODO align this with the webhook PR: https://github.com/kubernetes/kubernetes/pull/116261
+	// MatchConditions is a list of conditions on the AdmissionRequest that must be met
+	// for a request to be validated. All conditions must be met for the request to be matched.
+	// An empty list of matchConditions matches all requests. If there is an error evaluating the condition,
+	// the error is ignored and the condition is considered matched.
+	//
 	// +patchMergeKey=name
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=name
 	// +optional
```

Comment:
Mirror what we have in https://github.com/kubernetes/kubernetes/blob/c19c57ed5348517e8063ba1ea91cf6adc73765c1/staging/src/k8s.io/api/admissionregistration/v1/types.go#L311-L330 ?

Reply:
ack